week-5/schedule.md

```diff
@@ -0,0 +1,58 @@
+# Database Week Schedule
+
+## Day 1
+
+| Time    | Activity                                                                                                                                                                                      | Learning outcomes |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| 09:45   | Intro reading                                                                                                                                                                                 |                   |
+| 10:00   | [Intro presentation](https://docs.google.com/presentation/d/14LXEKmHM6xqjTvPLyKw0trtprGeNkD0VLgWkE8Z2ouo/edit#slide=id.g4dfce81f19_0_45)                                                      |                   |
+| 11:00   | [PSQL installation](https://github.com/macintoshhelper/learn-sql/blob/master/postgresql/setup.md) and [SQL commands intro workshop](https://github.com/foundersandcoders/sql-commands-intro/) | PSQL, SQL queries |
```

Comment:
Given the intro presentation shouldn't take more than 30 mins, I structured the other weeks to have the intro/installation stuff from 10:30-11:00. It doesn't really change the actual schedule, but could we do that here for consistency with the other weeks?

Reply:
yes will change that now